pygmt/tests/test_grdimage.py

```diff
@@ -51,3 +73,19 @@ def test_grdimage_fails():
     fig = Figure()
     with pytest.raises(GMTInvalidInput):
         fig.grdimage(np.arange(20).reshape((4, 5)))
+
+
+@pytest.mark.runfirst
+@pytest.mark.mpl_image_compare
+def test_grdimage_over_dateline(xrgrid):
```

Comment:
Running `test_grdimage_over_dateline` first (before the other tests) makes the tests pass on Linux (see commit 349904708048842f3c74a01b4a68cc42d62f5d18 and bf03e5b6deacd2f9e1c547762a483940d6df9770). This is a hacky workaround for the flakiness I described at https://github.com/GenericMappingTools/pygmt/pull/476/files#r453429872.